<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/imgs/BiliBili_favicon.ico</normaloff>:/imgs/BiliBili_favicon.ico</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_23">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab_my_manga">
      <attribute name="title">
       <string>我的漫画</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_19">
       <item>
        <layout class="QVBoxLayout" name="v_lLayout_my_manga">
         <item>
          <widget class="QTabWidget" name="tabWidget_my_manga">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab_manga_search">
            <attribute name="title">
             <string>搜索漫画</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_17">
             <item>
              <layout class="QHBoxLayout" name="h_Layout_manga_search" stretch="0,1,0,0">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>漫画名</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_manga_search_name">
                 <property name="maximumSize">
                  <size>
                   <width>300</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>别当哥哥了！</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_manga_search_name">
                 <property name="text">
                  <string>搜索</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_9">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QListWidget" name="listWidget_manga_search"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_local_storage">
            <attribute name="title">
             <string>本地库存</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <widget class="QListWidget" name="listWidget_local_storage"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_my_manga">
           <property name="title">
            <string>漫画信息</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item alignment="Qt::AlignTop">
             <widget class="QLabel" name="label_manga_image">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="resource.qrc">:/imgs/别当哥哥了.jpg</pixmap>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="v_Layout_manga_detail" stretch="0,0,0,1">
              <property name="spacing">
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="label_manga_title">
                <property name="text">
                 <string>标题：别当哥哥了！</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_manga_author">
                <property name="text">
                 <string>作者：猫豆腐，一迅社</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_manga_tag">
                <property name="text">
                 <string>标签：都市</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignTop">
               <widget class="QLabel" name="label_manga_outline">
                <property name="text">
                 <string>概要：  此漫画的翻译由版权方提供】游戏宅绪山真寻某天一觉醒来后发现自己变成女生了！原来是妹妹美波里为了进行观察实验而对他下药改造。于是真寻被迫开始了女生生活，他能否适应心身的改变，又是否可以继续当哥哥呢？  </string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_chp_detail">
         <property name="title">
          <string>章节详情</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
          <item>
           <layout class="QHBoxLayout" name="h_Layout_chp_detail">
            <item>
             <widget class="Line" name="line_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_chp_detail_total_chp">
              <property name="text">
               <string>总章节数：70</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_chp_detail_num_unlocked">
              <property name="text">
               <string>已解锁：10章</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_chp_detail_num_downloaded">
              <property name="text">
               <string>已下载：0章</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_chp_detail_num_selected">
              <property name="text">
               <string>已选中：15章</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_chp_detail_download_selected">
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>下载选中章节</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_chp_detail_download_all">
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>下载全部章节</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_chp_detail">
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="resizeMode">
             <enum>QListView::Adjust</enum>
            </property>
            <property name="viewMode">
             <enum>QListView::IconMode</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_download_list">
      <attribute name="title">
       <string>下载列表</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="h_Layout_total_progress">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="label_total_progress">
           <property name="text">
            <string>总进度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar_total_progress">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="value">
            <number>50</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget_download_list">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_processing">
          <attribute name="title">
           <string>进行中</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="v_Layout_processing" stretch="0,0">
             <item>
              <layout class="QHBoxLayout" name="h_Layout_processing">
               <property name="spacing">
                <number>10</number>
               </property>
               <item>
                <spacer name="horizontalSpacer_8">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_processing_start">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>开始选中任务</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_processing_pause">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>暂停选中任务</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_processing_delete">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>删除选中任务</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTableWidget" name="tableWidget_processing"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_finished">
          <attribute name="title">
           <string>已完成</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QVBoxLayout" name="v_Layout_finished" stretch="0,0">
             <item>
              <widget class="QTableWidget" name="tableWidget_finished"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="h_Layout_finished">
               <item>
                <spacer name="horizontalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_clear_tasks">
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>清理已完成任务</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_setting">
      <attribute name="title">
       <string>设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0,0,1,0,0">
       <item>
        <layout class="QHBoxLayout" name="h_Layout_my_cookie">
         <item>
          <widget class="QLabel" name="label_my_cookie">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>我的Cookie：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_my_cookie"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_my_cookie">
           <property name="text">
            <string>确认</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="h_Layout_save_path">
         <item>
          <widget class="QLabel" name="label_save_path">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>漫画保存路径：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_save_path"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_save_path">
           <property name="text">
            <string>浏览...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="h_Layout_num_thread">
         <item>
          <widget class="QLabel" name="label_num_thread_count">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>同时下载线程数：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="h_Slider_num_thread">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>36</number>
           </property>
           <property name="singleStep">
            <number>2</number>
           </property>
           <property name="pageStep">
            <number>2</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_num_thread">
           <property name="text">
            <string>线程数并不是越多越好，请根据自己的网络情况合理配置（推荐：8）</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="h_Layout_buttons">
         <property name="spacing">
          <number>50</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_open_log">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>打开日志</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_clear_data">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>清空用户数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_about">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>关于</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
